Practices/session_5_etl_and_airflow/README.md

```diff
@@ -0,0 +1,406 @@
+# ETL Introduction and Airflow
+
+In this practice we will develop a simple ETL pipeline on Airflow to understand each part of the ETL process
+
+![Docker-Python](documentation_images/etl-3.jpg)
+
+## Prerequisites
+
+* Follow the [pre-setup guideline][pre-setup]
+
+## Before start
+
+Let's review some concepts and technologies we used during the pre-setup:
+
+### ETL
+
+>ETL stands for `Extract, Transform, Load`, and it refers to the process of extracting data from one or more sources, transforming the data into a suitable format, and loading it into a target destination for storage or analysis.
+
+* `Extract` \
+  Data is extracted from one or more sources (databases, files, APIs...) \
+  It may be necessary to perform some data cleaning or filtering to ensure that only the relevant data is extracted.
+* `Transform` \
+  Extracted data is transformed into a suitable format for analysis or storage. \
+  This may involve cleaning the data, reformatting it, combining it with other data sources, or performing other data transformations.
+* `Load` \
+  Transformed data is loaded into a target destination.(database, a data warehouse, a data lake...)
+
+>Once the data is loaded it can be used for reporting, querying...
+
+![img](documentation_images/etl-2.png)
```

Comment:
Looks the image links are broken, no worries, just update those with new folder name "img/.."